docs/src/image.md

```diff
@@ -0,0 +1,20 @@
+# Image type syntax
+
+There are a huge number of combinations of image types in SPIR-V. They are represented by a const
+generic type called `spirv_std::image::Image`, however, specifying the generic parameters of this
+type is incredibly tedious, so a wrapper macro, `spirv_std::Image!` can be used to write the type
+instead.
+
+The specific syntax and meaning of the arguments to the `Image!` macro can be found in
+[rustdoc](https://embarkstudios.github.io/rust-gpu/api/spirv_std/macro.Image.html).
+
+Some type aliases for common image formats can be found in the `spirv_std::image` module. For
+example, `Image2d` is a very commonly used type, corresponding to `texture2D` in GLSL, and is likely
+what you want if you want a regular old sampled texture.
+
+```rust,no_run
+type Image2d = Image!(2D, type=f32, sampled);
+```
+
+Note that the `const-generics` feature must be enabled to use the `Image!` macro at the moment.
```

Comment:
It [looks like](https://play.rust-lang.org/?version=nightly&mode=debug&edition=2018&gist=001f2614aba60185e53c11f63baee177) the feature name is `const_generics`.

````suggestion
Note that the `const_generics` feature must be enabled to use the `Image!` macro at the moment.
```rust
#![feature(const_generics)]
// ...
```
````

Reply:
I see now that this is a cargo feature, so this should be a Cargo.toml example instead.